spec/requests/api/v1/push/subscriptions_spec.rb

```diff
@@ -56,19 +52,23 @@
     end
 
     it 'replaces old subscription on repeat calls' do
-      post :create, params: create_payload
+      post '/api/v1/push/subscription', params: create_payload, headers: headers
+
       expect(Web::PushSubscription.where(endpoint: create_payload[:subscription][:endpoint]).count).to eq 1
     end
 
     it 'returns the expected JSON' do
-      expect(body_as_json.with_indifferent_access).to include({ endpoint: create_payload[:subscription][:endpoint], alerts: {}, policy: 'all' })
+      expect(body_as_json.with_indifferent_access)
+        .to include(
+          { endpoint: create_payload[:subscription][:endpoint], alerts: {}, policy: 'all' }
+        )
     end
   end
 
-  describe 'PUT #update' do
+  describe 'PUT /api/v1/push/subscription' do
     before do
-      post :create, params: create_payload
-      put :update, params: alerts_payload
+      post '/api/v1/push/subscription', params: create_payload, headers: headers
```

Comment:
The setup here (both previous and my changes) is a little atypical -- using a POST request to make the data before the update or delete specs instead of just making the data directly -- I chose to just keep it the same for purposes of this PR to keep diff minimal. May review after merge to see if that's necessary or it can be a more typical setup.

Reply:
Yeah, it would probably make sense changing it in a follow-up PR, but it's perfectly fine as is for this PR!